homeassistant/components/light/flux_led.py

```diff
@@ -34,6 +34,8 @@
     vol.Optional(CONF_NAME): cv.string,
     vol.Optional(ATTR_MODE, default='rgbw'):
         vol.All(cv.string, vol.In(['rgbw', 'rgb'])),
```

Comment:
I agree. I need to get a true RGBWW light to test it with though. Most are just RGB(warm)W and not actually a 5th channel for the fancy white hue changing leds.